_posts/2019-05-16-let-the-dao-work.md

```diff
@@ -0,0 +1,36 @@
+title: "Let the DAO work"
+author: Manfred Karrer
+excerpt: "With the DAO in place Bisq has to evolve it's organisational structure without its founders. The departure of the founders was planned from the very beginning. Now it is time to fulfill that plan.<br><br>"
+---
+
+Dear Bisq community,
+
+With the successful launch of the DAO, Bisq is entering a new phase. The bootstrapping phase where founders are steering the project forward is over and the Bisq DAO is now the infrastructure which enables that Bisq is managed by its stakeholders and contributors. The DAO was envisioned from the very beginning of the project. It was also clear from the beginning that the founders need to step back to ensure that the project can develop to its full potential as a decentralised organisation. We see in all crypto-currency projects where the founder is still around that those projects suffers from some sort of centralisation. Satoshi did Bitcoin a great favour by stepping out. Bitcoin would not be the same if he would be still around as an authority, and Bitcoin developers would not be able to develop their full potential. Of course such events might be a bit disruptive and challenging for some but it really should be seen as that what it is: **a necessary step to enable Bisq to be a decentralized autonomous organization.**
+
+So with all that in mind I had planned to step back from most of my roles in Bisq after the DAO was launched. It was planned as a slow fade-out to make it as least disruptive as possible. Unfortunately this plan was interrupted by some exceptionally stressful event just around the time when we launched the DAO: Bisq got attacked by a chargeback scammer. Instead of celebrating and resting a bit after the quite exhaustive work to get the DAO ready for launch, I spent overtime on fighting the scammer. With the stress level that event caused it became clear to me that I have to take more care of myself to not risk any serious health issues or a burnout by constantly working over my limits. Knowing myself to be not very good in handling work-life balance I realised I cannot make my departure as a slow fade-out, but rather I need to make a hard cut, otherwise it would get delayed indefinitely. Of course I will be still around for helping and answering questions but I will step back from all my roles and activities in Bisq (beside a few trivial ones like running a Bitcoin node).
+
+So this moment might be a good occasion to look back to what we have achieved and look forward what is on our roadmap.
+
+- Bisq is the only really decentralised Bitcoin-Fiat exchange
+- We have developed the first Bitcoin based DAO
+- We built up a strong community of contributors
+- We earned a great reputation and Bisq became a quite well recognised brand in the Bitcoin community
+- Already more than 50% of Bisq traders are using BSQ already after just a few weeks. So the DAO is not only working on the contributors' side but also accepted by our traders.
```

Comment:
Add "is" before "also accepted by our traders."

Reply:
Thanks. What a sloppy writer I am ;-)